etherman/etherman.go

```diff
@@ -27,6 +28,10 @@ type IEtherman interface {
 	HeaderByNumber(ctx context.Context, number *big.Int) (*types.Header, error)
 	FilterSequenceBatches(opts *bind.FilterOpts,
 		numBatch []uint64) (*cdkvalidium.CdkvalidiumSequenceBatchesIterator, error)
+	WatchSetTrustedSequencer(opts *bind.WatchOpts,
```

Comment:
Not related with this PR, but can you rename this one from `IEtherman` to `Etherman`:

```go
//go:generate mockery --name IEtherman --output ../mocks --case=underscore --filename etherman.generated.go
```

Reply:
Fix: https://github.com/0xPolygon/cdk-data-availability/pull/43/commits/df45111433625aea768bd2706e2c1041bfd485e5